### DISCUSSION TITLE
Is there an ideal or better command setting to produce overall better footage for coloring?

### DISCUSSION THREAD

**Discussion by gregorpesek** [gopro/labs#520] [Q&A] [answered]
Hello,

I see there are few settings that adjust the way the footage comes out, for better post editing.
I would like to know which of the commands and settings produce the "best" or better looking footage then what is by default. Also is there any Rec.709 lut I could be using with those settings?

Also, what is this for: https://gopro.github.io/labs/build/

Cheers, Gregor

**Comment by dnewman-gpsw** [comment, marked as answer]
Better looking footage is subjective.  Depending on the scene the camera defaults are often great.  The main reason for some of the advanced controls is matching other cameras in a professional production, striving for a particular custom look, or dealing with complex lighting conditions.  

LOGB - Much can be done with the standard Flat profile, and EV compensation.  Labs extends Flat with LOGB, for a more aggressive Log curve.  LOGB=400 is likely the best, this is also what HERO12 is using with the new GP-Log.  For comparison standard Flat is LOGB=113.  When using LOGB you are emphasizing shadows and mid-tones, so you should underexpose, gaining highlight protections.  For Flat I run EV -0.5, for LOGB=400 I run EV -1 or -2 (GP-Log defaults to -2.) 

WIDE - As GoPro colors are saturated, and limited to a Rec709 gamut, it is easy to sharing in social media, youtube, and home TV. However there is more color out there than Rec709, and the WIDE=1 control, will stop targeting Rec709, and offer you the freedom to target other colorspaces with the wider gamut.  The camera has technically had this feature since Protune was added to HERO2, it is called White balance Native. However WIDE is easier to work with as you do not have to disable white balance.

BITR - Bitrate can be increased up to 200Mb/s, although 170Mb/s is stable for most video modes.  Using LOGB and WIDE, is adding more information, so you should add bitrate.

NR01 - Noise Reduction controls. LOGB will emphasize shadows, but the shadows have more noise. Normally the camera uses its advanced noise reduction to suppress the noise, but noise and shadow detail are hard to separate, so detail can be lost using the cameras defaults.  NR01=1 thru 100, is the percentage of NR to turn down, NR01=1 basically turns off the noise reduction. So you get more noise, and more details.  This require post noise reduction like using Neat Video to extract the best image.

There is more https://gopro.github.io/labs/control/extensions/ but the above are the key controls.

Re: https://gopro.github.io/labs/build/
Blockly is a visual scripting environment for creation advanced camera behaviors. Mostly used for creating custom camera triggers, but it can be used for scripting all Labs controls.

**Reply by gregorpesek** [reply]
Thank you very much!

With all those settings, do you use a basic LUT that you always apply to the clips?

I have neat video and I know how to use it, but it's quite tedious to work with and rather not. 
What is the default setting of NR01? 
Will NR01=1 make a huge difference in noise from what is the default? 
Does Neat Video preserve the details a lot better than how the GoPro processes it in camera?

Cheers!

**Reply by p0las** [reply]
I went through the "exercise" of trying to find the maximum quality settings for my 11. here are my findings, IDTs. LUTs etc:
https://nopixels.net/articles/unhappy-with-gopro-11-footage-try-this/
I hope this helps

**Reply by dnewman-gpsw** [reply]
@pOlas only item from your recommendation that I would change is the use for wN (white balance Native), and use !MWIDE=1 instead.  The WIDE feature is the same gamut (as native), but it corrects for white balance, you can use Auto or any of the fixed temperatures.

@gregorpesek NR01=0 is the default, all Labs features set to zero are removed from being active.  NR01=1 is the most noise/detail, and makes a difference in shadows.  NR01=50 is 50% of the noise reduction applied.  So you can go for a balance.  When the ISO is low, noise reduction can be off (I feel) and post noise reduction with Topaz or Neat Video is unnecessary.  Post noise reduction can use more advance AI models, and use more frames in the analysis than what can be performed in real-time in the camera, so for low light noisy footage post is best.

**Reply by gregorpesek** [reply]
@p0las Thank you for your findings.

@dnewman-gpsw Cheers. I just got back from a short road trip and there is a 1 second delay for the commands to start working when using quick capture, so the first second it looks everything to be on default and after the commands apply. Is there a fix for it?

Example: https://youtube.com/shorts/E9KMf9y47-o?feature=share

**Reply by dnewman-gpsw** [reply]
Currently there's no fix for that, Labs starts after the quick capture.  If you need labs features active throughout the capture you need to start the capture after normal boot.

**Comment by Pax709** [comment]
I did a lot of comparisons with GP11 and "wide" fared the worst, it creates artifacts and discoloration in color and you need to be aware of it. The second worst link is "Flat", which has GTM+LTM functions disabled , you lose fine detail irretrievably, it can not be recovered in any way. If someone is adjusting the material to another camera, he has no choice but to use Flat, others should stay with the setting Color Natural, NR, BITR. Very good results can be given by turning off LTM, the contrast in the shadows disappears slightly. If necessary, I can show you comparisons of what I described. Now I can see how much work has been done by Gopro developers and some things with H265 footage cannot be done better.
